src/modules/authorization/reducer.ts

```diff
@@ -74,9 +84,26 @@ export function authorizationReducer(
         data: [...state.data, ...authorizations]
       }
     }
+    case FETCH_AUTHORIZATION_SUCCESS: {
+      const { authorization } = action.payload
+      const data = state.data.map(stateAuthorization => {
+        if (authorization && areEqual(stateAuthorization, authorization)) {
+          return authorization
+        } else {
+          return stateAuthorization
+        }
+      })
```

Comment:
I'm not sure I understand this section of code, is it trying to replace the authorization with the new one?
What do you think about doing something like this?
```suggestion
      const data = state.data.filter(stateAuthorization => !areEqual(stateAuthorization, authorization))
      data.push(authorization)
```